examples/plot_simulate_somato.py

```diff
@@ -57,35 +64,63 @@
 # estimate the location and amplitude of a single current dipole. At the
 # moment, we do not offer explicit recommendations on which source
 # reconstruction technique is best for HNN. However, we do want our users
-# to note that the dipole currents simulate with HNN are assumed to be normal
-# to the cortical surface. Hence, using the option ``pick_ori='normal'``
-# seems to make most sense.
-
-method = "MNE"
+# to note that the dipole currents simulated with HNN are assumed to be normal
+# to the cortical surface. Hence, using the option ``pick_ori='normal'`` is
+# appropriate.
 snr = 3.
 lambda2 = 1. / snr ** 2
-stc = apply_inverse(evoked, inv, lambda2, method=method, pick_ori="normal",
-                    return_residual=False, verbose=True)
+evoked = epochs.average()
+stc = apply_inverse(evoked, inv, lambda2, method='MNE',
+                    pick_ori="normal", return_residual=False,
+                    verbose=True)
 
-pick_vertex = np.argmax(np.linalg.norm(stc.data, axis=1))
+###############################################################################
+# To extract the primary response in primary somatosensory cortex (S1), we
+# create a label for the postcentral gyrus (S1) in source-space
+hemi = 'rh'
+label_tag = 'G_postcentral'
```

Comment:
Hmm, ideally this would be the _sulcus_, `S_postcentral`. The visualisation is a bit harder, but it's more accurate in terms of modelling MEG signals. Is the visualisation your motivation to choose the gyrus instead?

Reply:
I tried `S_postcentral` first, however, the demarcation between 'gyrus' and 'sulcus' on the posterior wall of the CS (area 3b) is ambiguous. I found that using the 'sulcus' label did not include many of the highly responsive vertices of the inverse solution that should anatomically be considered part of area 3b.

Reply:
OK, this is a detail, fine for now.